WISE-Community/WISE
# Upgrade ThemeComponents/NavItem to Angular

Upgraded the component to Angular, downgrade to AngularJS so that it can be used in the AngularJS pages. Also de-coupled styling from AngularJS styles.

To test:
- Make sure nav items in unit plan (VLE) appear correctly, are styled.
- Make sure node icons all show up and are styled correctly across the apps (VLE, CM, AT).

Close #2801.
